http/readme.md

```diff
@@ -1,3 +1,42 @@
 # HyperText Transfer Protocol (HTTP)
 
+HTTP is a protocol **defines how client and server communicates**. 
```

Comment:
HTTP is a protocol `that` **defines how client and server communicates**. 

Maybe you can add a link to the Client-Server topic of this module! :D